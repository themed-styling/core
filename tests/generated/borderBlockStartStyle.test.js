import 'jest-styled-components'

import { borderBlockStartStyle } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(borderBlockStartStyle, 'borderBlockStartStyle')
