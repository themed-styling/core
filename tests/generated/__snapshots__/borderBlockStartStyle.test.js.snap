// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`borderBlockStartStyle renders 0 1`] = `
Array [
  .c0 {
  border-block-start-style: 0;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: 0;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: 0;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartStyle renders function 1`] = `
Array [
  .c0 {
  border-block-start-style: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: has;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: has;
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartStyle renders invalid theme path 1`] = `
Array [
  .c0 {
  border-block-start-style: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartStyle renders negative number 1`] = `
Array [
  .c0 {
  border-block-start-style: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartStyle renders object [breakpoints + standard] 1`] = `
Array [
  .c0 {
  border-block-start-style: July;
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block-start-style: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: July;
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block-start-style: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block-start-style: June;
  }
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block-start-style: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartStyle renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    border-block-start-style: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block-start-style: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    border-block-start-style: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block-start-style: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block-start-style: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-style: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block-start-style: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartStyle renders object [mediaQueries + standart] 1`] = `
Array [
  .c0 {
  border-block-start-style: July;
}

@media May {
  .c0 {
    border-block-start-style: August;
  }
}

@media April {
  .c0 {
    border-block-start-style: September;
  }
}

@media January {
  .c0 {
    border-block-start-style: November;
  }
}

@media April {
  .c0 {
    border-block-start-style: August;
  }
}

@media June {
  .c0 {
    border-block-start-style: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: July;
}

@media May {
  .c0 {
    border-block-start-style: August;
  }
}

@media April {
  .c0 {
    border-block-start-style: September;
  }
}

@media January {
  .c0 {
    border-block-start-style: November;
  }
}

@media April {
  .c0 {
    border-block-start-style: August;
  }
}

@media June {
  .c0 {
    border-block-start-style: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    border-block-start-style: November;
  }
}

@media April {
  .c0 {
    border-block-start-style: January;
  }
}

@media January {
  .c0 {
    border-block-start-style: December;
  }
}

@media April {
  .c0 {
    border-block-start-style: April;
  }
}

@media June {
  .c0 {
    border-block-start-style: May;
  }
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    border-block-start-style: November;
  }
}

@media April {
  .c0 {
    border-block-start-style: January;
  }
}

@media January {
  .c0 {
    border-block-start-style: December;
  }
}

@media April {
  .c0 {
    border-block-start-style: April;
  }
}

@media June {
  .c0 {
    border-block-start-style: May;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartStyle renders object [mediaQueries] 1`] = `
Array [
  @media May {
  .c0 {
    border-block-start-style: July;
  }
}

@media April {
  .c0 {
    border-block-start-style: August;
  }
}

@media January {
  .c0 {
    border-block-start-style: September;
  }
}

@media April {
  .c0 {
    border-block-start-style: November;
  }
}

@media June {
  .c0 {
    border-block-start-style: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    border-block-start-style: July;
  }
}

@media April {
  .c0 {
    border-block-start-style: August;
  }
}

@media January {
  .c0 {
    border-block-start-style: September;
  }
}

@media April {
  .c0 {
    border-block-start-style: November;
  }
}

@media June {
  .c0 {
    border-block-start-style: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    border-block-start-style: January;
  }
}

@media April {
  .c0 {
    border-block-start-style: December;
  }
}

@media January {
  .c0 {
    border-block-start-style: April;
  }
}

@media April {
  .c0 {
    border-block-start-style: May;
  }
}

@media June {
  .c0 {
    border-block-start-style: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    border-block-start-style: January;
  }
}

@media April {
  .c0 {
    border-block-start-style: December;
  }
}

@media January {
  .c0 {
    border-block-start-style: April;
  }
}

@media April {
  .c0 {
    border-block-start-style: May;
  }
}

@media June {
  .c0 {
    border-block-start-style: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartStyle renders postitive number 1`] = `
Array [
  .c0 {
  border-block-start-style: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartStyle renders string 1`] = `
Array [
  .c0 {
  border-block-start-style: July;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: July;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: June;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: June;
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartStyle renders undefined 1`] = `
Array [
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
]
`;

exports[`borderBlockStartStyle renders valid theme path 1`] = `
Array [
  .c0 {
  border-block-start-style: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-style: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
